styles.css

```diff
@@ -406,20 +393,14 @@ h1 {
 
```

Comment:
It looks like the `h1` color styling conflicts with your background styling! The text `Shares` disappears... 

From my understanding,`h1` tags have an effect on [SEO](https://www.greenlanemarketing.com/resources/articles/seo-101-seo-and-multiple-h1-tags/) & screen reader interpretation. There is a school of thought around only having one `h1` on your page, so you might consider making `Shares` a stylized `h2` with your darker font color.